src/Fable.Transforms/Rust/Fable2Rust.fs

```diff
@@ -1537,7 +1548,7 @@ module Util =
         then expr
         else expr |> makeLrcValue
 
-    let makeRecord (com: IRustCompiler) ctx r values entRef genArgs =
+    let makeRecord (com: IRustCompiler) ctx r isStruct values entRef genArgs =
```

Comment:
@alexswan10k Please remove change as `isCopyableEntity` already handles value types correctly.

Reply:
This is fine, I can remove it later.

Reply:
Oh I can move this logic inside. Let me fix..